ts/state/ducks/conversations.ts

```diff
@@ -268,8 +268,11 @@ export interface ReduxConversationType {
   isApproved?: boolean;
   didApproveMe?: boolean;
 
-  /** Should only be present on open groups - the key (stored as hex) that should be used when sending messages to an open group */
-  blindedPublicKey?: string;
+  // Should only be present on opengroups - the capabilities we have on this room.
+  capabilities?: Array<string>;
+  readCapability?: boolean;
+  writeCapability?: boolean;
+  uploadCapability?: boolean;
```

Comment:
Isn't reactions also a capability? Should we add it?

Reply:
capabilities is server wide here (blind, sogs, etc)
read/write and upload are room wide.

reactions is server wide, and so is part of the capabilities one